typedmodels/tests.py

```diff
@@ -444,3 +448,24 @@ class Tester2(Employee):
 
         class Tester3(Employee):
             name = models.IntegerField(null=True)
+
+
+def test_related_name_is_preserved_for_foreign_keys(db):
+    """Regression test for the following scenario:
+    A subclass of a typed model has foreign key to two models in a different app, while they are also related
+    to each other.
+    """
```

Comment:
```suggestion
    """
    Regression test for the following scenario:
    * A typed model has a ForeignKey or OneToOneField to a model in another app (lazily referenced via a string model name)
    * The ForeignKey or OneToOneField has a `related_name` specified.
    
    Previously, the related_name argument was not initialized correctly during startup, and threw an AttributeError.
    
     In this test, the fixed behaviour is tested via the `Order.order_review` and `Product.reviews` attributes.
    """
```


I'm still trying to get my head around what the issue is - is this an accurate description of what's going on?

Reply:
Is it important that the ForeignKey/OneToOneField is in another app, or would this happen with related models even in the same app if they used lazy/string references to each other?

Reply:
The description is accurate and indeed, the models being in a different app is what triggers the error. I encountered it in my project and this was the minimum demonstration possible.